src/components/post/Post.js

```diff
@@ -123,7 +124,7 @@ class Post extends Component {
             <Grid container justify="space-between" alignItems="flex-start">
               <Grid className="KnowledgePostText" item>
                 {upload.metadata.authors}
-                <div className="KnowledgePostTime">{upload.metadata.time}</div>
+                <div className="KnowledgePostTime">{upload.metadata.time || upload.metadata.createdAt}</div>
```

Comment:
what's the difference between time and createdAt?

Reply:
the endpoint to retrieve post when you click **showmore** only return createdAt